src/packet/signature.js

```diff
@@ -753,18 +753,19 @@ class SignaturePacket {
 export default SignaturePacket;
 
 /**
- * Creates a string representation of a sub signature packet
+ * Creates a Uint8Array representation of a sub signature packet
  * @see {@link https://tools.ietf.org/html/rfc4880#section-5.2.3.1|RFC4880 5.2.3.1}
  * @see {@link https://tools.ietf.org/html/rfc4880#section-5.2.3.2|RFC4880 5.2.3.2}
+ * @param {Boolean} critical - Whether the subpacket should be critical.
  * @param {Integer} type - Subpacket signature type.
  * @param {String} data - Data to be included
- * @returns {String} A string-representation of a sub signature packet.
+ * @returns {Uint8Array} The signature subpacket.
  * @private
  */
-function writeSubPacket(type, data) {
+function writeSubPacket(critical, type, data) {
```

Comment:
I think that `critical` should be passed after the type, otherwise the function calls are confusing to read

Reply:
Fair enough, changed :+1: